Model/Media.php

```diff
@@ -630,7 +629,7 @@ public function getCategory()
     /**
      * @param CategoryInterface $category|null
      */
-    public function setCategory(CategoryInterface $category = null)
+    public function setCategory($category = null)
```

Comment:
This commit is 2 years old and there are serveral tagged version between this and now, so it is a break.

https://3v4l.org/OvDJP